integration/kubernetes/runtimeclass_workloads/stress/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM quay.io/libpod/ubuntu:latest
+RUN apt-get -y update && \
+    apt-get -y upgrade && \
+    apt-get -y install stress
```

Comment:
Looking at the original `Dockerfile` (https://github.com/containerstack/alpine-stress/blob/master/Dockerfile) shows that `stress` originates from:

- https://people.seas.harvard.edu/~apw/stress.

That's confirmed by the debian package page too:

- https://packages.debian.org/en/buster/stress.

However, the homepage no longer exists (which presumably explains why sid doesn't list it - https://packages.debian.org/en/sid/stress).

I wonder if we want to use `stress-ng` instead?:

- https://packages.ubuntu.com/hirsute/stress-ng

Reply:
Hmmm, I did notice that, and I knew that stress-ng existed. There's also quay.io/kinvolk/stress-ng, which is at least also built for arm64, but not for the IBM architectures, so we'd probably still need the custom build. Let me TAL at how much we would have to change.

Reply:
Thanks @Jakob-Naucke. Hopefully, it should just be a case of changing the `apt` package name as `stress-ng` is tested on all arches fwics.

/cc @liubin and @GabyCT on the idea of switching to a different "stressor" package though.

Reply:
Yes, but I think the flags in the tests we use it in (`integration/kubernetes/k8s-{memory,oom}.bats`) will change.

Reply:
updated... it seems `stress-ng` is actually (and probably intentionally) forwards-comptaible.

Reply:
@Jakob-Naucke - nice! And thanks @ColinIanKing! :smile: